tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+from OMPython import OMParser
```

Comment:
why are you importing here and in the test_*.py files? is this a unittest quirk? (I'm mainly familiar with pytest)